### ISSUE TITLE
Contrasting Accessibility with Color Contrast by Todd Libby

### ISSUE THREAD

**Issue by meg-gutshall** [Virtual-Coffee/VC-Community-Docs#90]
Below is the info at hand for this brownbag event. Check the comments and project board status for any missing information. Items marked with an asterisk are required before publishing the final video. Attach any files in the bulleted list below or comments.

- [X] *Speaker name: Todd Libby
- [X] *Talk title: Contrasting Accessibility with Color Contrast
- [X] *Talk description: Accessibility is a huge issue these days. The latest WebAIM Million Report cited text contrast as the number one overlooked accessibility issue on the web today. Out of the 1 million sites tested, 86.3% had issues of text contrast that did not conform to WCAG standards. This talk delves into those issues and what we can do as designers and developers to fix this and start integrating accessibility in our workflows and projects from kickoff to handoff.
- [X] Date of the talk: 02/10/2021

- [ ] Speaker headshot:
- [X] Speaker bio: Todd Libby is a Web Developer/Designer and Accessibility Advocate. He has over 21 years of experience with the web in a multitude of different areas—JavaScript, ColdFusion, PHP, Rails, and Perl to name a few. He's into photography, travel, food, and cooking as a former chef with 20 years in the industry. Todd is a member of the W3C Silver Community Working Group and Visual Contrast of Text subgroup among several others.
- [ ] Speaker contact info:
  - [ ] Email address:
  - [ ] Website:
  - [ ] LinkedIn:
  - [ ] GitHub:
  - [ ] Twitter:
  - [ ] Other:

- [X] Speaker slides/notes:
    - [Contrasting Accessibility with Color Contrast.pdf](https://github.com/Virtual-Coffee/VC-Events/files/6053138/Contrasting.Accessibility.with.Color.Contrast.pdf)
- [X] Other resources shared during the talk:
    - [Accessible Color Palette Builder](https://toolness.github.io/accessible-color-matrix/)
    - [Color Safe](http://colorsafe.co/)
    - [EightShapes Contrast Grid](https://contrast-grid.eightshapes.com/)
    - [Tanaguru Contrast Finder](https://contrast-finder.tanaguru.com/)
    - [How to Use Color Blind Friendly Palettes to Make Your Charts Accessible](https://venngage.com/blog/color-blind-friendly-palette/)

- [ ] Intro card:
- [ ] YouTube privacy/visibility setting (pick one):
  - [ ] Published publicly
  - [ ] Published as unlisted
  - [ ] Published privately
